x/model/types/messages_model_test.go

```diff
@@ -1,104 +1,1064 @@
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.CommissioningCustomFlow = 1
+				msg.CommissioningCustomFlowUrl = ""
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "CommissioningCustomFlowUrl is set when CommissioningCustomFlow == 2",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.CommissioningCustomFlow = 2
+				msg.CommissioningCustomFlowUrl = "https://sampleflowurl.dclmodel"
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "CommissioningCustomFlowUrl length == 256",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.CommissioningCustomFlowUrl = "https://sampleflowurl.dclauth/" + tmrand.Str(256-30) // length = 256
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "CommissioningModeInitialStepsInstruction is omitted",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.CommissioningModeInitialStepsInstruction = ""
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "CommissioningModeInitialStepsInstruction length == 1024",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.CommissioningModeInitialStepsInstruction = tmrand.Str(1024)
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "CommissioningModeSecondaryStepsInstruction is omitted",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.CommissioningModeSecondaryStepsInstruction = ""
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "CommissioningModeSecondaryStepsInstruction length == 1024",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.CommissioningModeSecondaryStepsInstruction = tmrand.Str(1024)
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "UserManualUrl is omitted",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.UserManualUrl = ""
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "UserManualUrl length == 256",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.UserManualUrl = "https://sampleflowurl.dclauth/" + tmrand.Str(256-30) // length = 256
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "SupportUrl is omitted",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.SupportUrl = ""
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "SupportUrl length == 256",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.SupportUrl = "https://sampleflowurl.dclauth/" + tmrand.Str(256-30) // length = 256
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "ProductUrl is omitted",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.ProductUrl = ""
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "ProductUrl length == 256",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.ProductUrl = "https://sampleflowurl.dclauth/" + tmrand.Str(256-30) // length = 256
+				return msg
+			}(validMsgCreateModel()),
```

Comment:
Please add the following positive tests (for create and update messages):
- CommissioningModeInitialStepsHint=0
- CommissioningModeSecondaryStepsHint=0

Reply:
Added the tests for `MsgCreateModel`.
As to `MsgUpdateModel`, it does not have the specified fields.